docs/source/guides/logging.rst

```diff
@@ -1,7 +1,84 @@
-Request Logging
-===============
+Logging
+=======
 
-This part of the BentoML documentation is a work in progress. If you have any questions
-related to this, please join
-`the BentoML Slack community <https://join.slack.com/t/bentoml/shared_invite/enQtNjcyMTY3MjE4NTgzLTU3ZDc1MWM5MzQxMWQxMzJiNTc1MTJmMzYzMTYwMjQ0OGEwNDFmZDkzYWQxNzgxYWNhNjAxZjk4MzI4OGY1Yjg>`_
-and ask in the bentoml-users channel.
\ No newline at end of file
+BentoML uses standard `Python logging <https://docs.python.org/3/howto/logging.html>`_ libraries and provides basic logging customization through `bentoml.cfg` under the BentoML home directory. See below for configurable fields and default values.
+
+.. code-block:: init
+
+    [logging]
+    logging_config = {BENTOML_HOME}/logging.yaml
+
+    level = INFO
+    log_format = [%%(asctime)s] %%(levelname)s - %%(message)s
+    dev_log_format = [%%(asctime)s] {{%%(filename)s:%%(lineno)d}} %%(levelname)s - %%(message)s
+
+    # the base file directory where bentoml store all its log files
+    base_log_dir = {BENTOML_HOME}/logs/
+
+    log_request_image_files = True
+
+    prediction_log_filename = prediction.log
+    prediction_log_json_format = "%%(service_name)s %%(service_version)s %%(api)s %%(request_id)s %%(task)s %%(result)s %%(asctime)s"
+
+    feedback_log_filename = feedback.log
+    feedback_log_json_format = "%%(service_name)s %%(service_version)s %%(request_id)s %%(asctime)s"
+
+    yatai_web_server_log_filename = yatai_web_server.log    
```

Comment:
let's remove this line from here since yatai service logging is not really relevant here

Reply:
@yubozhao we should remove this config to under yatai_service section

Reply:
Talked to @yubozhao and will remove in a different PR.